spec/workers/metrics/record_data_counts_worker_spec.rb

```diff
@@ -14,6 +14,20 @@
 
     after { Rails.logger = default_logger }
 
+    it "calls count on each model" do
+      allow(User).to receive(:count)
+      described_class.new.perform
+      expect(User).to have_received(:count)
+    end
```

Comment:
Should we add
```ruby
expect(User).not_to have_received(:estimated_count)
```
since `count` is always called? I think this test would still pass even if it fell into the rescue block since it tries `count` first. Granted, that won't happen since in the test ENV. So this question is more for my understanding.